include/intersectables/lightGeometry.h

```diff
@@ -2,10 +2,15 @@
 #define LIGHT_GEOMETRY_H
 
 #include "../hitRecord.h"
+#include "boundingBox.h"
 #include "intersectable.h"
 
 class LightGeometry : public Intersectable {
  public:
   virtual HitRecord* sample() const = 0;
+  BoundingBox getBoundingBox() const {
+    BoundingBox foo(Vector3f(0, 0, 0), Vector3f(0, 0, 0));
```

Comment:
dummy implementation. Not sure what we should return here. Maybe, some kind of `isInvalid` value for bounding boxes should exist. Maybe this also indicates that `getBoundingBox()` does not belong to the **Intersectable** interface.